### DISCUSSION TITLE
Einweisung als unnötig markieren

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#76] [Shipped - Live on production]
Manche Mitarbeiter:innen sind zwar in Gruppen, brauchen aber nicht auf alle Geräte der Gruppe eine Einweisung. 
Eine Markierung, dass die Einweisung für diese Person auf dieses Gerät nicht notwendig ist wäre wünschenswert.

**Comment by YvesR** [comment]
Coming with https://github.com/Samedis-care/samedis-care-issues/issues/1244
